linux_pipeline/Jenkinsfile_azure_new_vm_size_provision

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env groovy
+def RunPowershellCommand(psCmd) {
+    bat "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\""
+}
+
+
+env.NewVMsEnv = ""
+env.todaysSizes = ""
+env.current = ""
+env.CurrentSize =  ""
+env.CurrentRegion =  ""
+env.NewVMSizeEmailBody = "SR\t\tSize\t\tRegion\n"
+env.test_cmd= "BVT-VERIFY-DEPLOYMENT-PROVISION"
+stage("Checkout")
+{
+    timeout(120) {
+        node("master")
```

Comment:
don't run this stage on master, run it on ubuntu metaslave

Reply:
Fixed